app/components/github/project-repo-sync.js

```diff
@@ -0,0 +1,60 @@
+import Component from '@ember/component';
+import { get, computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+  classNames: ['github__project-repo-sync'],
+
+  projectGithubRepo: null,
+  totalSteps: 7,
```

Comment:
I think a common thing would be to declare this as a const up top, outside the object. 

Either

```Javascript
const TOTAL_STEPS = 7;

export default Component.extend({
  totalSteps: TOTAL_STEPS,
});
```
or even just using the `const` directly instead of re-assigning it to `this.`

Reply:
Another thought I had was pulling this from the API rather than hardcoding it, a la Stripe's `fields_needed`. That's way overdoing it, though.

Another minor refactor in that direction could be putting the steps in a simple array of objects as a `const` outside the component object.

Reply:
that works as well, I actually prefer that